docs/en_US/Installation.md

```diff
@@ -24,6 +24,29 @@ Currently we only support installation on Linux & Mac.
 
   You can also install NNI in a docker image. Please follow the instructions [here](https://github.com/Microsoft/nni/tree/master/deployment/docker/README.md) to build NNI docker image. The NNI docker image can also be retrieved from Docker Hub through the command `docker pull msranni/nni:latest`.
 
+## **Installation on Windows**
+
+* __Install NNI through pip__
+
+  Prerequisite: `python >= 3.5`
+  ```bash
+  python -m pip install --upgrade nni
+  ```
+
+* __Install NNI through source code__
+
+  Prerequisite: `python >=3.5, git, powershell`
+  When you use powershell to run script for the first time, you need run powershell as Administrator with this command:
+  ```bash
+  Set-ExecutionPolicy -ExecutionPolicy Unrestricted
+  ```
+  Then you can install nni as administrator or current user as follows:
+  ```bash
+  git clone https://github.com/Microsoft/nni.git
```

Comment:
Why we install source code on the master branch instead of a branch of a certain version? For example, Linux installation use v0.6. Version branch is stable

Reply:
Because windows local mode does not belong to any version branch now. Maybe change it to v0.7 later.

Reply:
ok. Got it.